File: Selenium/Activities/Activity20.py
from selenium import webdriver
from selenium.webdriver.common.by import By



with webdriver.Firefox() as driver:
	
    driver.get("https://training-support.net/webelements/alerts")
    print(f"The page title is : {driver.title}")
			
    btnPrompt =  driver.find_element(By.CSS_SELECTOR,"button#prompt")
    result = driver.find_element(By.CSS_SELECTOR,"p#result")
			
    btnPrompt.click();
    alert = driver.switch_to.alert
    print(f"The text on the simple alert is : {alert.text}")
    alert.send_keys("Awesome")
    alert.accept()
    print(f"Result message is : {result.text}")
    driver.quit()

File: Selenium/Activities/Activity20.java
package examples;

import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.Alert;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;

public class Activity20 {

	public static void main(String[] args) {

			WebDriver driver = new FirefoxDriver();
			
			driver.get("https://training-support.net/webelements/alerts");
			System.out.println("The page title is : "+ driver.getTitle());
			
			WebElement btnPrompt =  driver.findElement(By.cssSelector("button#prompt"));
			WebElement result = driver.findElement(By.cssSelector("p#result"));
			
			btnPrompt.click();
			
			Alert alert = driver.switchTo().alert();
			System.out.println("The text on the simple alert is : "+alert.getText());
			alert.sendKeys("Awesome");
			alert.accept();
			System.out.println("Result message is : " + result.getText());
				driver.quit();

	}

}
